plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/configuration/DebugConfigurationAction.java

```diff
@@ -53,14 +51,15 @@ public DebugConfigurationAction(DebugConfigurationsManager debugConfigurationsMa
     public void updateInPerspective(ActionEvent event) {
         Optional<DebugConfiguration> configurationOptional = configurationsManager.getCurrentDebugConfiguration();
         if (configurationOptional.isPresent() && configuration.equals(configurationOptional.get())) {
-            event.getPresentation().setText(TICK + configuration.getName());
+            event.getPresentation().setVisible(false);
```

Comment:
event.getPresentation().setVisible(<expression>)
instead of 
if (<expression>) {
event.getPresentation().setVisible(false)
} else {
event.getPresentation().setVisible(true)
}